Assets/Scripts/Player/PlayerMover.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using UnityEngine;
+
+[RequireComponent(typeof(Player))]
+public class PlayerMover : MonoBehaviour
+{
+    [SerializeField] private float _speed;
+    [SerializeField] private float _oneBoostTime;
+
+    private Player _player;
+    private int _speedFactor = 1;
+
+    private void Awake()
+    {
+        _player = GetComponent<Player>();
+    }
+
+    private void OnEnable()
+    {
+        _player.BoosterRaised += Boost;
+    }
+
+    private void OnDisable()
+    {
+        _player.BoosterRaised -= Boost;
+    }
+
+    private void Update()
+    {
+        Move(GetDirection());
+    }
+
+    private void Boost()
+    {
+        StartCoroutine(BoostSpeed());
+    }
+
+    private IEnumerator BoostSpeed()
+    {
+        _speedFactor++;
+        yield return new WaitForSeconds(_oneBoostTime);
+        _speedFactor--;
+    }
+
+    private void Move(Vector3 direction)
+    {
+        transform.Translate(direction * _speed * _speedFactor * Time.deltaTime);
+    }
+
+    private Vector3 GetDirection()
+    {
+        float directionX = Input.GetAxis("Horizontal");
+        float directionY = Input.GetAxis("Vertical");
+
```

Comment:
https://trello.com/c/UAeMGORw/17-%D0%BE%D1%82%D0%BA%D0%B0%D0%B7-%D0%BE%D1%82-%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%BE%D0%B2%D1%8B%D1%85-%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B5%D0%BB%D0%BE%D0%B2